.stylelintrc

```diff
@@ -1,6 +1,7 @@
 {
   "extends": [
     "@lmc-eu/stylelint-config",
-    "@lmc-eu/stylelint-config/optional"
+    "@lmc-eu/stylelint-config/optional",
+    "stylelint-config-prettier"
```

Comment:
Do you think this should be part of our stylelint config?

Reply:
We use prettier and its recommended to use this config in [@lmc-eu/stylelint-config](https://github.com/lmc-eu/code-quality-tools/tree/main/packages/stylelint-config). I had to add it otherwise this PR didn't pass tests (stylelint was colliding with prettier).